### ISSUE TITLE
iOS support?

### ISSUE THREAD

**Issue by petarjs** [triniwiz/nativescript-videorecorder#5]
Hello, is this an Android only plugin? I tried adding it to iOS NativeScript project, I get this error:

```
CONSOLE LOG file:///app/main-view-model.js:43:24: TypeError:   
UIVideoEditorController.availableMediaTypesForSourceType is not a function. (In   
'UIVideoEditorController.availableMediaTypesForSourceType(sourceType)',   
'UIVideoEditorController.availableMediaTypesForSourceType' is undefined)  
```

**Comment by triniwiz** [comment]
No ios support was added by @victorsosa

**Comment by triniwiz** [comment]
@petarslovic the error was fixed update to `1.2.2`
